dimod/generators/wireless.py

```diff
@@ -0,0 +1,988 @@
+                concatenated with an imaginary vector.
+
+            * '16QAM'
+
+                Each user is assumed to select independently from 16 symbols.
+                The transmitted symbol is a complex value that can be encoded 
+                by two spins in the imaginary part and two spins in the real 
+                part. Highest precision real and imaginary spin vectors are 
+                concatenated to lower precision spin vectors.
+
+            * '64QAM'
+
+                A QPSK symbol set is generated and symbols are further amplitude 
+                modulated by an independently and uniformly distributed random 
+                amount from :math:`[1, 3]`.
+
+            * '256QAM'
+
+                A QPSK symbol set is generated and symbols are further amplitude 
+                modulated by an independently and uniformly distributed random 
+                amount from :math:`[1, 3, 5]`.
+
+        y: Complex- or real-valued received signal, as a NumPy array. If 
+            ``None``, generated from other arguments.
+
+        F: Complex- or real-valued channel, as a NumPy array. If ``None``, 
+            generated from other arguments. Note that for correct interpretation
+            of SNRb, channel power should be normalized to ``num_transmitters``.
+
+        num_transmitters: Number of users. Each user transmits one symbol per 
+            frame.
+
+        num_receivers: Number of receivers of a channel. Must be consistent 
+            with the length of any provided signal, ``len(y)``.
+
+        SNRb: Signal-to-noise ratio per bit used to generate the noisy signal 
+            when ``y`` is not provided. If ``float('Inf')``, no noise is
+            added. 
+            
+            :math:`SNR_b = E_b/N_0`, where :math:`E_b` is energy per bit, 
+            and :math:`N_0` is the one-sided power-spectral density. :math:`N_0` 
+            is typically :math:`k_B T` at the receiver. To convert units of 
+            :math:`dB` to :math:`SNR_b` use :math:`SNRb=10^{SNR_b[decibels]/10}`.
+
+        seed: Random seed, as an integer, or state, as a 
+            :class:`numpy.random.RandomState` instance.   
+            
+        transmitted_symbols:
+            Set of symbols transmitted. Used in combination with ``F`` to 
+            generate the received signal, :math:`y`. The number of transmitted 
+            symbols must be consistent with ``F``.
+            
+            For BPSK and QPSK modulations, statistics of the ensemble do not 
+            depend on the choice: all choices are equivalent. By default, 
+            symbols are chosen for all users as :math:`1` or :math:`1 + 1j`, 
+            respectively. Note that for correct analysis by some solvers, applying 
+            spin-reversal transforms may be necessary.
+            
+            For QAM modulations, amplitude randomness affects likelihood in a 
```

Comment:
Sometimes QPSK is referred to as 4QAM, and this makes sense since they are just conventions. However, statements like this that relate to QAM but not QPSK then risk misleading. not sure of the simplest solution though.

Reply:
I updated to "For QAM modulations such as 16QAM, amplitude randomness affects..."

Reply:
In context of this function, where QPSK is clearly distinguished from QAM there is no issue anyway.